backend/chainlit/config.py

```diff
@@ -257,9 +263,13 @@ class UISettings(DataClassJsonMixin):
     custom_js: Optional[str] = None
     custom_font: Optional[str] = None
     # Optional custom meta tag for image preview
-    custom_meta_image_url: Optional[str] = None
+    custom_og_image_url: Optional[str] = None
+    # Optional custom meta url
+    custom_url: Optional[str] = None
     # Optional custom build directory for the frontend
     custom_build: Optional[str] = None
+    # Optional custom Twitter card image URL
+    twitter_card_image_url: Optional[str] = None
```

Comment:
If I understand it well, you added these new configuration options. In this first PR, I would recommend against changing anything.

Rather, in the second PR, I would suggest a _single_ configuration option for setting arbitrary headers. 

The reason is that otherwise we end up with a large, possibly infinite number of specific configuration options, which don't really save that much work as the actual templated HTML code isn't that complex. We'd keep the originals only for compatibility.

But I'm open to other ideas, what do you think?

Reply:
So, you're suggesting something like this -
`arb_header : Optiona[dict]`

And then in the `get_html_template` function, we can unwrap and convert this `arb_header` dict into html and pass it to index.html.

Will this work ? 
I believe this will allow for infinite expansion of headers, as and when required. This will also keep the code minimum.

Reply:
I have removed the twitter social card, as suggested to keep the config file same as original. Kindly suggest if i should proceed with arb_headers ?